test/integration/targets/vmware_vsan_health_info/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/vcenter
+shippable/vcenter/group1
```

Comment:
We won't be able to run the test in our CI because of the VMware vSAN Management SDK dependency. Could you replace `shippable/vcenter/group1` by `unsupported`.

Reply:
Done